cves/kernel/CVE-2014-3144.yml

```diff
@@ -1,199 +1,171 @@
-  For one-line strings, you can just use quotes after the colon
-  For multi-line strings, as we do for our instructions, you put a | and then
-  indent by two spaces
-
-  For readability, we hard-wrap multi-line strings at 80 characters. This is
-  not required, but appreciated.
-curated_instructions: |
-  If you are manually editing this file, then you are "curating" it.
-
-  Set the version number that you were given in your instructions.
-
-  This will enable additional editorial checks on this file to make sure you
-  fill everything out properly. If you are a student, we cannot accept your work
-  as finished unless curated is properly updated.
-curation_level: 0
-reported_instructions: |
-  What date was the vulnerability reported to the security team? Look at the
-  security bulletins and bug reports. It is not necessarily the same day that
-  the CVE was created.  Leave blank if no date is given.
-
-  Please enter your date in YYYY-MM-DD format.
-reported_date:
-announced_instructions: |
-  Was there a date that this vulnerability was announced to the world? You can
-  find this in changelogs, blogs, bug reports, or perhaps the CVE date.
-
-  This is not the same as published date in the NVD - that is below.
-
-  Please enter your date in YYYY-MM-DD format.
-announced_date: '2014-05-11'
-published_instructions: |
-  Is there a published fix or patch date for this vulnerability?
-  Please enter your date in YYYY-MM-DD format.
-published_date: '2014-05-11'
-description_instructions: |
-  You can get an initial description from the CVE entry on cve.mitre.org. These
-  descriptions are a fine start, but they can be kind of jargony.
-
-  Rewrite this description IN YOUR OWN WORDS. Make it interesting and easy to
-  read to anyone with some programming experience. We can always pull up the NVD
-  description later to get more technical.
-
-  Try to still be specific in your description, but remove project-specific
-  stuff. Remove references to versions, specific filenames, and other jargon
-  that outsiders to this project would not understand. Technology like "regular
-  expressions" is fine, and security phrases like "invalid write" are fine to
-  keep too.
-
-  Your target audience is people just like you before you took any course in
-  security
-description:
-bounty_instructions: |
-  If you came across any indications that a bounty was paid out for this
-  vulnerability, fill it out here. Or correct it if the information already here
-  was wrong. Otherwise, leave it blank.
-bounty:
-  amt:
-  announced:
-  url:
-reviews: []
-bugs_instructions: |
-  What bugs are involved in this vulnerability?
-
-  Please list bug IDs to https://bugzilla.kernel.org/
+CWE:
+  - 191
+ipc:
+  note: This vulnerability caused the system to crash causing a denial of service. It is less of a direct feature that is an interprocess communication more so that any ipc would be killed by the crash.
```

Comment:
How? What lead to the crash? A null pointer? A "normal" person wouldn't be able to read this and understand

Reply:
I will fix this. I believe it is a null pointer. At initial read of the bug it just says it causes a system crash but after looking at it I will add a note about null pointer and why.